src/Razor/src/Microsoft.VisualStudioCode.RazorExtension/Services/RazorWorkspaceService.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor.Features;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.VisualStudioCode.RazorExtension.Services;
+
+[ExportRazorStatelessLspService(typeof(RazorWorkspaceService)), Shared]
+[method: ImportingConstructor]
+file class WorkspaceService(ILoggerFactory loggerFactory) : RazorWorkspaceService
```

Comment:
File name doesn't match class name. Also not sure why this is a file class. I guess it doens't hurt, but seems odd.